src/data/observability-packs.json

```diff
@@ -0,0 +1,535 @@
+ {
+  "observabilityPacks": [
+    {
+      "authors": [
+        "New Relic",
+        "Jakub Kotkowiak"
+      ],
+      "dashboards": [
+        {
+          "description": "",
+          "name": "Apache",
+          "screenshots": [
+            "https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.3.1/packs/apache/dashboards/apache01.png",
+            "https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.3.1/packs/apache/dashboards/apache02.png",
+            "https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.3.1/packs/apache/dashboards/apache03.png"
+          ],
+          "url": "https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.3.1/packs/apache/dashboards/apache.json"
+        }
+      ],
+      "alerts": [
+        {
+          "name": "Response Time",
+          "definition": "",
+          "url": ""
+        },
+        {
+          "name": "Throughput",
+          "definition": "",
+          "url": ""
+        }
+      ],
+      "description": "Official New Relic pack for Apache\nUse this pack together with the New Relic Apache On Host Integration to get insight into the performance of your Apache instances.\n\n",
+      "icon": "https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.3.1/packs/apache/icon.png",
+      "id": "QXBhY2hl",
+      "level": "NEWRELIC",
```

Comment:
should we change the level on a couple of these? maybe add a `COMMUNITY` or two (not actually sure of the official terms)

Reply:
Good call! I'll add a few of the other levels